pages/price-feeds/migrate-an-app-to-pyth/chainlink.md

```diff
@@ -90,6 +90,8 @@ contract PythAggregatorV3Deployment is Script {
 
 ```
 
+Check the [Chainlink Migration Example](https://github.com/pyth-network/pyth-examples/tree/main/price_feeds/evm/chainlink_migration) for a complete example of how to deploy the `PythAggregatorV3` contract.
```

Comment:
I am not happy with the words "complete example". Can't think of any other at the moment.

Reply:
How about "for a runnable version of the example code above"?  

nitpick: I don't think "Check" is the right verb here. I think "Please see" is the best option, or possibly "Consult" or "Refer to".